### PR TITLE
Ejecting out of cryo

### PR DESCRIPTION
**Pull request by Jaygriff**
Adds verb to eject out of cryo

### CODE REVIEW

File: code/game/machinery/cryo.dm [hunk 1]
```diff
@@ -76,19 +76,33 @@
...
-		return
-	sleep(usr.stat * 1200)
-	open_machine()
-	return
+/obj/machinery/atmospherics/unary/cryo_cell/verb/move_eject()
+	set name = "Eject Cryo Cell"
+	set desc = "Begin the release sequence inside the cryo tube."
+	set category = "Object"
+	set src in oview(1)
+	if(usr == occupant || contents.Find(usr))	//If the user is inside the tube...
```
Comment (nalarmg) [inline code/game/machinery/cryo.dm:91]: Redundant contents.Find. Only one person can be in a cryogenics tube at a time, and if they're not the occupant then they're not in the tube.

File: code/game/machinery/cryo.dm [hunk 2]
```diff
@@ -76,19 +76,33 @@
...
+		if(usr.stat == DEAD)	//and he's not dead....
+			return
+		usr << "<span class='notice'>Release sequence activated. This will take about a minute.</span>"
+		spawn(700)
+			if(!src || !usr || (!occupant && !contents.Find(usr)))	//Check if someone's released/replaced/bombed him already
```
Comment (nalarmg) [inline code/game/machinery/cryo.dm:96]: Redundant contents.Find. Only one person can be in a cryogenics tube at a time, and if they're not the occupant then they're not in the tube.

**Comment by Nokington** [comment]
DNM until the things Nalar pointed out are addressed

**Comment by Jaygriff** [comment]
Tg port directly. Will run tests tomorrow

**Pull request closed without merging**